README.md

```diff
@@ -27,7 +28,11 @@ Proudly brought to you by [@revolunet](http://twitter.com/revolunet), [@deltaeps
  - offline mode
  - analytics.js advanced debugging support
 
-## Installation
+## Installation and Quick Start
+You need to read rest of documentation, but here is the quick start.
```

Comment:
This sentence doesn't provide a lot of value to the reader. It may be better to explain the quick start. Perhaps:

"The quick start is designed to give you a simple, working example for the most common usage scenario. There are numerous other ways to configure and use this library as explained in the documentation."